PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/referencebook/AzureSdkArtifactGroupPanel.java

```diff
@@ -109,4 +125,41 @@ private void createUIComponents() {
         this.toolbar.setForceMinimumSize(true);
         this.toolbar.setTargetComponent(this.viewer);
     }
+
+    /**
+     * referred com.intellij.application.options.schemes.AbstractSchemesPanel.ShowSchemesActionsListAction
+     *
+     * @see com.intellij.application.options.schemes.AbstractSchemesPanel
+     */
+    private static class ShowDependencyTypeListAction extends DefaultActionGroup {
+        private String selected;
```

Comment:
selected is not a good name for options: maven or gradle

Reply:
we have two fields: selected and type to represent gradle or maven, is it a bit duplicate?